bin/test

```diff
@@ -1,12 +1,17 @@
 #!/bin/sh
 
-# Set test variables
-export DATABASE_URL=mysql2://root:secret@mysql/test
+# We use TEST_DATABASE_URL to make sure we don't accidentaly overwrite dev/prod db
+unset DATABASE_URL; export DATABASE_URL
```

Comment:
To avoid overwriting the current database when TEST_DATABASE_URL is not set but DATABASE_URL is (e.g. in a Docker container). In that case config/database.yml is used when present, or an error is given.